custom-prompt/custom-prompt.c

```diff
@@ -282,7 +282,8 @@ void display_primary_prompt(char const *git_info)
 {
     char const *venv = getenv("VIRTUAL_ENV_PROMPT");
     LOG_DEBUG("Current Python virtual environment is '%s'.", venv);
-    LOG_DEBUG("Showing primary prompt.");
+    int shlvl = atoi(getenv("SHLVL"));
+    LOG_DEBUG("Current shell level is %d.", shlvl);
```

Comment:
Should `SHLVL` be passed as a command line argument?

Reply:
In MSYS2, the shell level is reported as 0 in the debug logs, but `echo $SHLVL` displays `1`. Furthermore, the shell level isn't correctly detected in WSL, since the program is a Windows executable.

Reply:
Also, use `strtol` for consistency. That's what I've used elsewhere.